hack/generator/pkg/astmodel/storage_conversion_function.go

```diff
@@ -220,25 +205,12 @@ func (fn *StorageConversionFunction) generateBody(
 	parameter string,
 	generationContext *CodeGenerationContext,
 ) []dst.Stmt {
-	if fn.intermediateType == nil {
-		// Last step of conversion, directly working with the hubType type we've been given
-		switch fn.conversionDirection {
-		case ConvertFrom:
-			return fn.generateDirectConversionFrom(receiver, parameter, generationContext)
-		case ConvertTo:
-			return fn.generateDirectConversionTo(receiver, parameter, generationContext)
-		default:
-			panic(fmt.Sprintf("unexpected conversion direction %q", fn.conversionDirection))
-		}
-	}
-
-	// Intermediate step of conversion, not working directly with the hubType type we've been given
-	// Instead we convert to/from our intermediate type (which is one step closer to the hub type in our conversion graph)
+	// Last step of conversion, directly working with the hubType type we've been given
```

Comment:
is `hubType` in this comment correct?

Reply:
No it's not, and I don't think we need any comment here now. Removed.